### ISSUE TITLE
ADD FALSE POSITIVES HERE 

### ISSUE THREAD

**Issue by alganzory** [alganzory/HaramBlur#64]
If you encounter a false positive (something that's labeled haram but shouldn't be) please follow the following steps:

1. add a link to the media or a screenshot (you can use any image upload service but don't post it here directly so people don't accidentally see what they don't want to see)
2. write your settings (strictness, gender) and browser
3. if possible, please add your pc specs or at least let us know if it's a high-end/lower-end device

Check the example below

**Comment by alganzory** [comment]
1. [Image Link](https://www.google.com/url?sa=i&url=https%3A%2F%2Fwww.shutterstock.com%2Fsearch%2Fblack-woman-face&psig=AOvVaw2MfjUGHHfzVdR7z31GBA_l&ust=1707855079463000&source=images&cd=vfe&opi=89978449&ved=0CBMQjRxqFwoTCJDdtYzOpoQDFQAAAAAdAAAAABAJ)
2. 40^ strictness, female gender selected, chrome
3. High-end laptop

**Comment by EmberCraze** [comment]
1. Media: [Image Link](https://images.confetticdn.com/images/scale/w_1000,f_auto/event-98593/image-f8e1860a-23e8-4eb5-b1e2-1ad8cb1b6f46/google-mingel.jpeg)
2. Settings: 100% strictness, female gender selected, firefox
3. Device: high end laptop

**Comment by mr1lazycoder** [comment]
assalamo alaykom, your addon is a lifesaver i just want to say that it would be very beneficial if you could add a way to report false positives and false negatives directly through the add on i don't know maybe by including two buttons next to the images and somehow it uploads them or send the link to the image directly to you. it's been like three days i have a high end pc and i use firefox, i thought the addon was disabled since it stopped bluring images even though i have it on 100% strictness females only, a simple google search for the term "women" did blur some images but many are still unblured. baraka allaho fik i hope you update it soon

**Comment by bibo242** [comment]
I agree with what @mr1lazycoder said, we need like a shortcut that automatically uploads the false negative/ false positive mage if we right click on it or something because that is way faster to train the model. I think the data about the users pc (high end / low end ) should be saved in the extension also. With the way reporting false negatives work, it is very tedious for someone to go out of his way and report it.

**Comment by Zaituny** [comment]
1. Media: https://ibb.co/Pvcb8SjZ
2. Settings: 100% strictness, female gender, Chrome
3. device: high-end laptop

**Comment by Raviu56** [comment]
1. Thumbnail of https://www.youtube.com/shorts/1gmeNl2xhoQ 
2. Settings: up to 80% , female, Firefox
3. High-end PC

1. Thumnail of https://www.youtube.com/watch?v=5dUVbAGNHn4
2. Settings: up to 100% , female, Firefox
3. High-end

**Comment by Raviu56** [comment]
100% strictness, high-end PC
Preface/Thumbnail of those youtube videos:
https://www.youtube.com/watch?v=WQeoO7MI0Bs
https://www.youtube.com/watch?v=2FYm3GOonhk
https://www.youtube.com/watch?v=sz25xxF_AVE

**Comment by primatelab** [comment]
1. Media: [Protest For Women's Rights In Afghanistan](https://themedialine.org/wp-content/uploads/2023/08/GettyImages-1593600579-e1692131454932.jpg)
2. Settings: 100% strictness, female gender selected, firefox
3. Device: high end laptop
4. Reason: One of the women is not entirely invisible behind her burqa. Also none of them are being beaten or beheaded.

---

1. Media: [https://www.hrw.org/topic/womens-rights](https://www.hrw.org/topic/womens-rights) 
2. Settings: 100% strictness, female gender selected, firefox
3. Device: high end laptop
4. Reason: Entire page is haram.

**Comment by mohamed91700sth** [comment]
Media : https://www.scan-vf.net/one_piece/chapitre-711
Settings: 100% strictness, female gender selected, Chrome
Device: high end laptop
